mitosheet/src/mito/components/modals/UserEditedCodeModal.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Mito
+
+import React, { Fragment } from 'react';
+import { MitoAPI } from '../../api/api';
+import { UIState, UserProfile } from '../../types';
+import DefaultModal from '../DefaultModal';
+import TextButton from '../elements/TextButton';
+import { ModalEnum } from './modals';
+
+/*
+    This modal displays to the user when:
+    1. the analysis that they are replaying does not exist on their computer
+    2. the analysis errors during replay for some other reason
+*/
+const UserEditedCodeModal = (
+    props: {
+        setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+        mitoAPI: MitoAPI,
+        userProfile: UserProfile,
+        onClickButton: (overwriteUserEdits: boolean) => void,
+    }): JSX.Element => {
+    return (
+        <DefaultModal
+            header='Insert New Cell?'
```

Comment:
I think we could change the header to something less action oriented and more communicative about _why_ the user is there. Let's change this to `Edit to Code Detected`?